_posts/2020-02-19-js-the-language-meetup.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "JavaScript The Language:  Async/Await"
+date: 2020-02-19 10:00:00 +0200
+categories: javascript
+author_names: ["Pablo Palacios", "Sergii Paryzhskyi"]
+read_time : 5
+show_related_posts: false
+square_related:
+feature_image: posts/2020-02-19-js-the-language-meetup-async-await/poster.jpg
+---
+
+Yesterday we've hosted a “JavaScript the Language” Meetup at HolidayCheck. This is a monthly meetup where we go into details about specific aspects of JavaScript. The topic of Thursday's meetup was Async/Await.
```

Comment:
Later you're using the same wording "Async/await", but with lower case "async/await". I would suggest sticking to one form. Or maybe even `async`/`await` (or simpler `async/await`) using code syntax to emphasize that it's a language construct?